junebug/command_line.py

```diff
@@ -64,7 +64,7 @@ def parse_arguments(args):
     parser.add_argument(
         '--inbound-message-ttl', '-ittl', dest='inbound_message_ttl', type=int,
         help='The maximum time allowed to reply to a message. Defaults to '
-        '"60" seconds')
+        '60*10 seconds or 10 minutes')
```

Comment:
I think we need to reword this to say something like `The maximum time allowed to reply to a message (in seconds). Defaults to 600 seconds (10 minutes)`, otherwise it becomes unclear what the units are.